src/views/other-qualification-details.njk

```diff
@@ -0,0 +1,157 @@
+{% extends "_base.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+
+{% set otherDateErrors %}
+  {% if model.otherDayError %}
+    <p>The date you obtained your qualification must include a day</p>
+  {% elif model.otherMonthError %}
+    <p>The date you obtained your qualification must include a month</p>
+  {% elif model.otherYearError %}
+    <p>The date you obtained your qualification must include a year</p>
+  {% elif model.otherFutureError %}
+    <p>The date you obtained your qualification must be in the past</p>
+  {% else %}
+    <p>The date you obtained your qualification must be a real date</p>
+  {% endif %}
+{% endset -%}
+
+{% block content %}
+
+  {% if model.otherQualificationError %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+            text: "Enter the Name of your certificate",
+            href: "#other-name-error"
+        } if model.otherNameError,
+        {
+            text: "The date you obtained your qualification must be in the past",
+            href: "#other-date-error"
+        } if model.otherFutureError,
+        {
+            text: "The date you obtained your qualification must be a real date",
+            href: "#other-date-error"
+        } if model.otherInvalidError,
+                        {
+            text: "The date you obtained your qualification must include a day",
+            href: "#other-date-error"
+        } if model.otherDayError,
+        {
+            text: "The date you obtained your qualification must include a month",
+            href: "#other-date-error"
+        } if model.otherMonthError,
+        {
+            text: "The date you obtained your qualification must include a year",
+            href: "#other-date-error"
+        } if model.otherYearError,
+        {
+            text: "You must provide qualification details",
+            href: "#other-date-error"
+        } if model.emptyPageError
+      ]
+    }) }}
+  {% endif %}
+
+  <h1 class="govuk-heading-xl">
+    What are the details of your Other/Equivalent deer stalking certificate?
+  </h1>
+  <p class="govuk-body">
+    Please add the name and the date obtained of the certificate you use to apply
```

Comment:
https://design-system.service.gov.uk/components/error-message/#be-clear-and-concise

Don't say `please`!

Also, this whole sentence should probably be re-worded - it's a bit 'stumbly' at the moment.

Reply:
Agreed to be honest I think the sentence can just be removed and change the heading to 

```Enter the details of your Other/Equivalent deer stalking certificate below``` 

this keeps the same pattern as the DSC 1 and 2 page

Reply:
Sounds good to me - nice to see a consistent approach.

Resolved in 1111a1c.